testrunner/execute.go

```diff
@@ -230,6 +232,16 @@ func removeObsoleteParentTests(tests []testResult) []testResult {
 	return results
 }
 
+func formatTestNames(tests []testResult) []testResult {
+	var out []testResult
```

Comment:
```suggestion
	out := make([]testResult, 0, len(tests))
```

This makes a slice with 0 elements, but with capacity to grow up to `len(tests)` before more memory allocations are needed. Makes the function a little bit more memory-allocation friendly.